packages/serverless-components/aws-cloudfront/test-utils.js

```diff
@@ -25,7 +25,9 @@ module.exports = {
       expect.objectContaining({
         DistributionConfig: expect.objectContaining({
           CacheBehaviors: expect.objectContaining({
-            Items: [expect.objectContaining(cacheBehavior)]
+            Items: expect.arrayContaining([
```

Comment:
Would you prefer this is a separate helper? i copied the `arrayContaining` approach from your origins test.

Reply:
I think it seems fine, we are anyway trying to not manage AWS manually in the future (instead using CDK or something similar)